From bb9c424f0b52bafff4785bb4a69c585a8204c4c9 Mon Sep 17 00:00:00 2001
From: ShevT <shev_t@mail.ru>
Date: Wed, 13 Jun 2018 15:21:54 +0300
Subject: [PATCH] AndroidManifest: Remove duplicates uses-permissions

Remove duplicates:
android.permission.CHANGE_WIFI_STATE
android.permission.ACCESS_WIFI_STATE

Change-Id: If4d1e6ab08f99dee16f1efb1b727d402387cfa4b
---
 AndroidManifest.xml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 5b25c3c1..15c301f4 100755
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -44,8 +44,6 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.MANAGE_USERS" />
     <uses-permission android:name="android.permission.INTERNAL_SYSTEM_WINDOW" />
-    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
-    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.SET_ACTIVITY_WATCHER" />
     <uses-permission android:name="android.permission.CONNECTIVITY_INTERNAL" />
     <uses-permission android:name="android.permission.NFC_HANDOVER_STATUS" />
-- 
2.17.1

